src/main/java/com/softserve/academy/spaced/repetition/service/MailService.java

```diff
@@ -69,4 +76,12 @@
      */
     String getRestorePasswordTemplateContent(Map<String, Object> model);
 
+    /**
+     * Generates the text of email with details of user's request which we get via contact us form.
+     *
+     * @param model map with values which will be inserted into html page.
+     * @return content of the email.
+     */
+    String contactUsMailTemplateContent(Map<String, Object> model);
```

Comment:
As I understand this method is a private method of service implementation (MailServiceImpl.java). There is no need to expose it through the interface.
If something can be private - make it private (encapsulation).